views/partials/dashboards/remove-form-confirm.hbs

```diff
@@ -21,25 +23,36 @@
         <span class="remove-dashboard-confirm-item-label">{{getRemoveString
             "remove-dash-confirm-label-full"
           }}</span>
-        <span class="remove-dashboard-confirm-item-entry"></span>
+        <span class="remove-dashboard-confirm-item-entry">
+          {{#if this.acctInfo.names.0.first_name}}{{this.acctInfo.names.0.first_name}}{{/if}}
+          {{#if this.acctInfo.names.0.middle_name}}{{this.acctInfo.names.0.middle_name}}{{/if}}
+          {{#if this.acctInfo.names.0.last_name}}{{this.acctInfo.names.0.last_name}}{{/if}}
+        </span>
       </li>
       <li class="remove-dashboard-confirm-item" data-id="account">
         <span class="remove-dashboard-confirm-item-label">{{getRemoveString
             "remove-dash-confirm-email"
           }}</span>
-        <span class="remove-dashboard-confirm-item-entry"></span>
+        <span class="remove-dashboard-confirm-item-entry">
+          {{#if this.acctInfo.emails.0.email}}{{this.acctInfo.emails.0.email}}{{/if}}
+        </span>
       </li>
       <li class="remove-dashboard-confirm-item" data-id="location">
         <span class="remove-dashboard-confirm-item-label">{{getRemoveString
             "remove-dash-confirm-loc"
           }}</span>
-        <span class="remove-dashboard-confirm-item-entry"></span>
+        <span class="remove-dashboard-confirm-item-entry">
+          {{#if this.acctInfo.addresses.0.city}}{{this.acctInfo.addresses.0.city}}{{/if}}, 
```

Comment:
If `this.acctInfo.addresses.0.city` is falsy, I think this will have an orphaned comma here. Not sure if the comma should be moved within the `{{#if }}..{{/if}}` block.

Reply:
Hi Peter! Nice one!